scrum/tools/bin/get_custom_field_ids.py

```diff
@@ -0,0 +1,32 @@
+""" Print the custom fields object of the first card with values populated
+  Useful for reviewing structure and determining config values in case of a
+  new board or a new custom field
+"""
+
+## Get SP from current system
+import os, sys, pprint
```

Comment:
Update for consistency with other scripts - i.e. individual imports, or move all files doing import os,sys etc into one line